Build break on 1.4.3 branch

**Issue Description**
Build break on 1.4.3 branch.
While cherry picking issue #6155 to 1.4.3 I noticed a compiulation error in a file that is not impacted by that cherry-pick 

```
/home/progier/sb/t1.4.3/tst/source/389-ds-base/ldap/servers/slapd/entry.c: In function 'slapi_entry_attr_get_valuearray':
/home/progier/sb/t1.4.3/tst/source/389-ds-base/ldap/servers/slapd/entry.c:2894:34: error: returning 'struct slapi_value **' from a function with incompatible return type 'const struct slapi_value **' [-Wincompatible-pointer-types]
 2894 |     return attr->a_present_values.va;

```

A clear and concise description of what the bug is.

**Package Version and Platform:**
 - Platform: Fedora
 - Package and version:  remotes/origin/389-ds-base-1.4.3

**Steps to Reproduce**
Steps to reproduce the behavior:
1. clone 389-ds-base
2. switch to 389-ds-base-1.4.3 branch
3. Try to build the rpms
4. See error

**Expected results**
No errors

Sigh! I get another error with make -f rpm.mk rpms 
```
ldap/servers/slapd/pw_verify.c: In function ‘pw_verify_token_dn’:
ldap/servers/slapd/pw_verify.c:110:16: warning: initialization discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
  110 |     char *dn = slapi_sdn_get_dn(sdn);
      |                ^~~~~~~~~~~~~~~~
ldap/servers/slapd/pw_verify.c:114:9: error: implicit declaration of function ‘fernet_verify_token’ [-Wimplicit-function-declaration]
  114 |     if (fernet_verify_token(dn, cred->bv_val, key, tok_ttl) != 0) {
      |         ^~~~~~~~~~~~~~~~~~~
make[2]: *** [Makefile:11603: ldap/servers/slapd/ns_slapd-pw_verify.o] Error 1
```

--- a/ldap/servers/slapd/entry.c
+++ b/ldap/servers/slapd/entry.c
@@ -2891,7 +2891,7 @@ slapi_entry_attr_get_valuearray(const Slapi_Entry *e, const char *attrname)
         return NULL;
     }
 
-    return attr->a_present_values.va;
+    return (const struct slapi_value **)(attr->a_present_values.va);
 }
 
 /*

--- a/ldap/servers/slapd/pw_verify.c
+++ b/ldap/servers/slapd/pw_verify.c
@@ -26,6 +26,8 @@
 #include "slap.h"
 #include "fe.h"
 
+int fernet_verify_token(const char *dn, const char *token, const char *raw_key, uint64_t ttl);
+
 int
 pw_verify_root_dn(const char *dn, const Slapi_Value *cred)
 {

--- a/ldap/servers/slapd/slap.h
+++ b/ldap/servers/slapd/slap.h
@@ -16,6 +16,13 @@
 #include <config.h>
 #endif
 
+/*
+ * Make sure to define statvfs struct before defining f_type macro
+ * otherwise there may be a conflict
+ */
+#include <sys/statvfs.h>
+
+
 /* slap.h - stand alone ldap server include file */
 
 #ifndef _SLDAPD_H_

--- a/src/lib389/setup.py
+++ b/src/lib389/setup.py
@@ -13,7 +13,8 @@
 #
 
 from setuptools import setup, find_packages
-from os import path
+from os import path, environ
+from sys import argv
 import build_manpages as bm
 if bm.__version__ < '2.1':
     from build_manpages import build_manpages as bm
@@ -29,6 +30,18 @@ version = "1.4.0.1"
 
 with open(path.join(here, 'README.md'), 'r') as f:
     long_description = f.read()
+
+#
+# For some historical reason when using prefix install
+#  files that should be in /usr/sbin/, /usr/share/ are
+#  in $PREFIX/sbin, $PREFIX/share
+# So lets mimick this behavior
+
+prefix=environ.get('INSTALL_PREFIX', '/usr')
+print(f"argv={argv}")
+if prefix != "/usr" and 'install' in argv and '--prefix' not in argv:
+    argv.append('--prefix')
+    argv.append(prefix)
 
 setup(
     name='lib389',
@@ -59,19 +72,19 @@ setup(
 
     # find lib389/clitools -name ds\* -exec echo \''{}'\', \;
     data_files=[
-        ('/usr/sbin/', [
+        (prefix+'/sbin/', [
             'cli/dsctl',
             'cli/dsconf',
             'cli/dscreate',
             'cli/dsidm',
             ]),
-        ('/usr/share/man/man8', [
+        (prefix+'/share/man/man8', [
             'man/dsctl.8',
             'man/dsconf.8',
             'man/dscreate.8',
             'man/dsidm.8',
             ]),
-        ('/usr/libexec/dirsrv/', [
+        (prefix+'/libexec/dirsrv/', [
             'cli/dscontainer',
             ]),
     ],

